test/system/projects/samples_test.rb

```diff
@@ -704,5 +704,21 @@ class SamplesTest < ApplicationSystemTestCase
       end
       assert_text I18n.t('projects.samples.attachments.deletions.destroy.success')
     end
+
+    test 'user should not see sample attachment delete buttons if they are non-owner' do
```

Comment:
This test should be broken up into 2 different tests (one for when the user can see the delete buttons and one for when the user can't)

Reply:
Fixed in 8c4be64cd61a4a7fae8b642efa2a2c6c75d0ea7f